scenarios/19c_The_Oracle.cfg

```diff
@@ -1305,9 +1489,49 @@
                 description=_"Death of Lady Lorin"
                 condition=lose
             [/objective]
+            {TURNS_RUN_OUT}
+            {OBJECTIVE_NOTES _"The scenario will continue even if Matthias Ramon dies; it is the keep upon which he stands that requires greater protection."}
```

Comment:
You can just keep the first part of the clause.

Reply:
changed in 61ba872